runtime/j9vm/java11vmi.c

```diff
@@ -690,6 +692,20 @@ allowReadAccessToModule(J9VMThread * currentThread, J9Module * fromModule, J9Mod
 	return retval;
 }
 
+/* Convert '.' to '/' in the package name. */
+static void
+convertPackageName(char* packageName)
+{
+	char ch = '\0';
+	char* p = (char*)packageName;
+	while(0 != (ch = *p)) {
+		if ('.' == ch) {
+			*p = '/';
+		}
+		p++;
+	}
+}
```

Comment:
I expect build failures where this function is unused (jdk8-14).

* the cast in the initialization of `p` is redundant
* style: assignment in `while` condition
* inconsistent NUL char `'\0'` vs. `0`
* missing space after `while`

Reply:
Assignment in a while is allowed, referring to the OMR coding standards.
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md
`Never combine an assignment and a test, except in a while condition.`

Reply:
OK, I wasn't aware of that exception.